python/spacewalk/server/importlib/backendLib.py

```diff
@@ -494,15 +494,7 @@ def query(self, values):
 
 def sanitizeValue(value, datatype):
     if isinstance(datatype, DBstring):
-        if value is None or value == "":
-            return None  # we really want to preserve Nones
-            # and not depend on Oracle converting
-            # empty strings to NULLs -- PostgreSQL
-            # does not do this
-        if len(value) > datatype.limit:
-            value = value[: datatype.limit]
-            # ignore incomplete characters created after truncating
-        return value
+        return _sanitize_dbstring(value, datatype)
```

Comment:
I refactored this into a separate function so that sonarcloud won't fail with method complexity being too high.